compiled_starters/java/src/main/java/Main.java

```diff
@@ -20,8 +20,8 @@ public static void main(String[] args){
           byte[] header = Files.readAllBytes(Path.of(databaseFilePath));
 
           // The page size is stored at the 16th byte offset, using 2 bytes in big-endian order.
-          // '& 0xFFFF' is used to convert the signed short to an unsigned int.
-          int pageSize = ByteBuffer.wrap(header).order(ByteOrder.BIG_ENDIAN).position(16).getShort() & 0xFFFF;
+          short pageSizeSigned = ByteBuffer.wrap(header).order(ByteOrder.BIG_ENDIAN).position(16).getShort();
```

Comment:
Do we need `.order(ByteOrder.BIG_ENDIAN)`? 
[Doc](https://docs.oracle.com/javase/8/docs/api/java/nio/ByteBuffer.html) says that ByteBuffer is BIG_ENDIAN by default:
> The initial order of a byte buffer is always BIG_ENDIAN.

Reply:
Yep, I think the initial intention here was to be more explicit - but yeah safe to remove now that you mention it